CookingRecipies/Component/Login.js

```diff
@@ -0,0 +1,90 @@
+import React, {useState} from 'react';
+import {
+  StyleSheet,
+  View,
+  Text,
+  TextInput,
+  TouchableOpacity,
+} from 'react-native';
+import axios from 'axios'
+
+
+
+const Login = () => {
+const [login, SetLogin] = useState({username: '', password: ''})
+
+console.log(login)
+
+const onPress = () => {
+ console.log("axios call goes here")
+  // axios.put('', login)
+  // .then(res => console.log('response from login axios post', res))
+  // .catch(err => console.log('error from login axios post',err))
+}
+
+  return (
+    <View style={styles.signUp}>
+          <Text>Login</Text>
+          <TextInput
+           style={styles.inputFeilds}
+           placeholder="Username"
+           name="username"
+           value={login.username}
+           onChangeText={event => SetLogin({...login, username:event})}/>
```

Comment:
by convention, the setter state variable first letter is lowercased.  like setLogin